submissions/Ant-C-tech/tiny-js-world/index.js

```diff
@@ -0,0 +1,99 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Ant-C-tech/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Spike',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof-Woof!',
+   friends: ['John']
+}
+
+const cat = {
+   species: 'cat',
+   name: 'Tom',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Meow-Meow!',
+   friends: ['John']
+}
+
+const woman = {
+   species: 'human',
+   name: 'Jane',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Nice to meet you!',
+}
+
+const man = {
+   species: 'human',
+   name: 'John',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi, how are you?',
+   friends: ['Spike', 'Tom']
+}
+
+function createGreetingContent(obj) {
+
+   let content = ''
+
+   const propPrefixes = {
+      species: 'I am a %. ',
+      name: 'My name is %. ',
+      gender: 'My gender is %. ',
+      hands: 'I have % hands. ',
+      legs: 'I have % legs. ',
+      saying: 'I want to say you "%". ',
+      friends: 'I have friend %. ',
+   };
+
+   Object.keys(obj).forEach((key) => {
+      if (Array.isArray(obj[key])) {
+         for (const item of obj[key]) {
+            content += propPrefixes[key].replace('%', item)
+         }
+      } else if (obj[key] && propPrefixes[key]) {
+         content += propPrefixes[key].replace('%', obj[key])
+      }
+   })
```

Comment:
Order of keys in an object is not guaranteed. Whenever a particualr order is required avoid `Object.values|keys|entries` methods as well as `for ... in` or `for ... of`.
List propertis to publish explicitly. This way you will also have an opportunity to skip any properties.

When processing arrays string concatenation is an anti-pattern. Employ `Array.map` and `Array.join`. This way you will get rid of unnecessary `content` variable and `.trim()`.